package.json

```diff
@@ -7,21 +7,28 @@
     "typecheck": "tsc"
   },
   "dependencies": {
+    "@ariakit/react": "^0.2.3",
+    "@radix-ui/react-label": "^2.0.1",
     "@remix-run/node": "^1.7.2",
     "@remix-run/react": "^1.7.2",
     "@remix-run/vercel": "^1.7.2",
+    "@remix-run/web-fetch": "^4.3.4",
```

Comment:
When creating a PR, please be mindful of what's in your `package.json`. I don't think this package is used anywhere?

Reply:
This package helped me with an error related to my Node version (v.20) which appeared after the "npm run dev" command. It seems that all project members should use the same Node versions. I can add it to .gitignore.